bin/manuscripts

```diff
@@ -36,12 +36,14 @@ from dateutil import parser
 sys.path.insert(0, '.')
 from manuscripts.report import Report
 from manuscripts.config import Config
+from manuscripts._version import __version__
 
 def get_params():
     """Parse command line arguments"""
 
     parser = argparse.ArgumentParser()
-
+    parser.add_argument('-v', '--version', action='version', version=__version__,
```

Comment:
It seems the comment on `version` (to change it to `default`) is not yet addressed.... Any comment about the reason?

If there is no specific reason, please read the docs (you had the link in the previous review) and change version to use `default` instead.

By the way, I think `action` is not needed, because if none is set, it will be `version` by default, because the argument is `--version`. So, please remove it (again, except if you have a good reason for this).

Thanks!

Reply:
As I understand it, -v/--version is to be used to check the version information of manuscripts. Wouldn't making it `default=` instead of `version=` convert it into a command-line argument which defaults to a value if the argument is not given rather than just giving the version of manuscripts whenever -v/--version is used ?
I was trying to do something similar to what is given [here](https://docs.python.org/3/library/argparse.html#action) by using `action="version"`.

(Sorry for posting the same thing as in the last review)

Reply:
I'm sorry. I was totally confused. Thanks for the explanation. Yes, now I think this is the way to go.